packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -793,7 +809,7 @@ export const SDP = {
   MAX_FS_8160: 'max-fs=8160',
   B_LINE: 'b=TIAS',
   CARRIAGE_RETURN: '\r\n',
-  BAD_MEDIA_PORTS: [9, 0]
+  BAD_MEDIA_PORTS: [0]
```

Comment:
What was the cause of this change?

Reply:
9 is actually a valid port . It gets generated at start of the call . Previously it was not . Only 0 is bad port now